build/run-e2e-tests-policy-framework-prow.sh

```diff
@@ -27,13 +27,20 @@ echo "* Install cert manager"
 $DIR/install-cert-manager.sh
 
 echo "* Set up cluster for test"
-# Set tag for images: Use `latest` for `main` and `latest-<version>` for `release-<version>` branches
-# `PULL_BASE_REF` is a variable provided by Prow: 
-# https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables
-export VERSION_TAG="latest"
-if [ "${PULL_BASE_REF}" ] && [ "${PULL_BASE_REF}" != "main" ]; then
-  export VERSION_TAG="${VERSION_TAG}-${PULL_BASE_REF#*-}"
+
+# Set tag for images: Use `latest` for `main` and `latest-<version>` for `release-<version>`
+# branches. If the PR is in `openshift/release`, parse the job spec for the target branch.
+# Otherwise, use `PULL_BASE_REF`.
+VERSION_TAG="latest"
+if [[ "${REPO_OWNER}" == "openshift" ]] && [[ "${REPO_NAME}" == "release" ]]; then
+  TARGET_BRANCH="$(echo "${JOB_SPEC}" | jq -r '.extra_refs[] | select(.workdir == true).base_ref')"
+else
+  TARGET_BRANCH="${PULL_BASE_REF}"
 fi
+if [ "${TARGET_BRANCH}" ] && [ "${TARGET_BRANCH}" != "main" ]; then
+  VERSION_TAG="${VERSION_TAG}-${PULL_BASE_REF#*-}"
+fi
```

Comment:
Nit: I know it was single `[` before, but since the new part uses `[[`, I'd rather use that consistently. Unless there's a good reason?

I think you've made and maintained more of these scripts than I have, so if you have another style guide then lmk, but I usually go to https://google.github.io/styleguide/shellguide.html

Reply:
Nope, no style guide--that looks like a good reference, thanks! I definitely prefer consistency, and `[[` certainly is much nicer to deal with. The one minor thing is that it's my understanding that `[` is POSIX compliant while `[[` is not, but in this case it probably doesn't matter. I'll update it. `[[` fo life! 😆